app/views/attach-pvc.html

```diff
@@ -113,6 +113,32 @@ <h2 class="text-center">No persistent volume claims.</h2>
                       </div>
 
                       <div class="form-group">
+                        <label for="sub-path">Subpath</label>
+                        <input
+                            id="sub-path"
+                            class="form-control"
+                            type="text"
+                            name="subPath"
+                            ng-model="attach.subPath"
+                            placeholder="example: application/resources"
+                            ng-pattern="RELATIVE_PATH_PATTERN"
+                            autocorrect="off"
+                            autocapitalize="off"
+                            spellcheck="false"
+                            aria-describedby="sub-path-help">
+                        <div id="sub-path-help" class="help-block">
+                          Optional path within the volume from which the container's volume should
+                          be mounted. Defaults to the volume's root.
```

Comment:
Maybe:
Optional path within the volume from which it will be mounted into the container. Defaults to the volume's root.